anda/terra/obsolete/terra-obsolete.spec

```diff
@@ -121,6 +121,8 @@ BuildArch:  noarch
 %obsolete_ticket https://github.com/terrapkg/packages/issues/991
 %obsolete iosevka-fusion-fonts 25.1.1-2
 
+%obsolete_ticket https://github.com/terrapkg/packages/pull/2522
+%obsolete terra-gnome-shell-extension-pop-shell 1.2.0^20241010.e25621e-2
```

Comment:
EVR (epoch, version, release) needs to be strictly higher than the last EVR of the package

Reply:
That defeats the purpose though, I specifically want to obsolete the current release of the package, not future versions, which won't get built anyway when the spec is removed?